docs/changelog/82409.yaml

```diff
@@ -0,0 +1,19 @@
+pr: 82409
+issues:
+  - 81210
+  - 52728
+area: Search
+type: enhancement
+summary: Allow doc-values only search on number types
+highlight:
+  notable: false
+  title: Allow doc-values only search on number types
```

Comment:
Not sure what the difference is between summary and title, but both should reflect now that doc-values only search is supported on numeric, date, keyword, ip, and boolean type.

Reply:
`summary` is for the PR, and appears in the list of changes document (`docs/reference/release-notes/8.1.asciidoc`). `highlight.title` goes into the release highlights document (`docs/reference/release-notes/highlights.asciidoc`).